### DISCUSSION TITLE
no options to increase vram

### DISCUSSION THREAD

**Discussion by Simonghost117** [DavidS95/Smokeless_UMAF#56] [Q&A]
On my laptop, hp probook with ryzen 5 4500u, no option appears in the "device list"

**Comment by Monokel777** [comment]
With my HP Probook x360 435 G8 with an AMD 5800U I don't have any options either.

Do we have to deactivate any "security"BIOS-Options?

greetings

**Comment by GitGivi** [comment]
On my laptop, hp 15-ba060nl, no graphic option appears in the "device list".

**Comment by Espionage724** [comment]
I can increase video RAM with Intel UHD 630 (`lspci` shows something go from `256M` to `2G` with a related BIOS setting; apeture?)

I think Above 4G decode and something else needed enabled too.
